test/test_animalInfo.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from app import app
+
+@pytest.fixture
+def client():
+    app.config['TESTING'] = True
+    client = app.test_client()
+    yield client
```

Comment:
You shouldn't need any of these imports or the pytest fixture. See my other comment about moving the new endpoint inside of the `create_app` function